tests/validation/cattlevalidationtest/core/test_github.py

```diff
@@ -60,13 +60,16 @@ def config():
 @pytest.fixture(scope='module')
 def github_request_code(config, cattle_url, admin_client, request, user=None):
     def fin():
-            admin_client.create_githubconfig(enabled=False)
+            admin_client.create_githubconfig(enabled=False,
+                                             accessMode='restricted')
```

Comment:
James, does this change mean that there is no default value for accessMode? As before we weren't passing anything for this param.